tests/integration/scoping/src/library.cc

```diff
@@ -43,6 +43,23 @@ void validate(legate::TaskContext& context)
   }
 }
 
+void map_check(legate::TaskContext& context)
+{
+  int32_t task_count          = context.get_launch_domain().get_volume();
+  int32_t shard_id            = context._get_shard_id();
```

Comment:
can we use `legate::Processor::get_executing_processor().address_space()` instead? we should stop exposing Legion details to the API